examples/Jobs/JobsSample/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿#pragma warning disable CS0618 // Type or member is obsolete
+using System.Text;
+using Dapr.Jobs;
+using Dapr.Jobs.Extensions;
+using Dapr.Jobs.Models.Responses;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddDaprJobsClient();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+
+app.MapDaprScheduledJob("myJob", (ILogger logger, JobDetails details) =>
+{
+    logger.LogInformation("Received trigger invocation for 'myJob'");
+
+    if (details.Payload is not null)
+    {
+        var deserializedPayload = Encoding.UTF8.GetString(details.Payload.Value.ToArray());
```

Comment:
We offer extension methods for serializing objects/strings when scheduling jobs; should we have equivalent ones for use in deserializing them from a handler (e.g. `details.PayloadAsJson<TJsonType>()`?

Reply:
Oh, I see, the extensions were added to `byte[]`.  I feel like we might want to make it more job-details-specific in order to be more discoverable.  Also, we might want to avoid, for example, customers using those extension methods as generic byte-serializing/deserializing methods.